src/integration/HubspotApiModel.ts

```diff
@@ -63,6 +63,74 @@ export class HubspotApiModel {
             return JSON.parse(e.response.body);
         }
     }
+    /**
+     * 
+     * @param objectType object to create property to check property for 
+     * @param propertyName the name of the property, not the label
+     * @returns true if the property exists, otherwise false
+     */
+    public async objectHasProperty(objectType: string, propertyName: string){
+        const readPropertyURL = 'https://api.hubapi.com/crm/v3/properties/' + objectType + '/' + propertyName; 
+        try {
+         const response = await axios.get(readPropertyURL, {
+             params: {
+                 hapikey: this.HAPIKEY
+             }
+         });
+            if (response.status==200 ){
+                console.log("=== PROPERTY EXIST IN OBJECT TYPE " + objectType + " ===")
```

Comment:
Looks good! Maybe we can remove the console.logs for the success cases, and only log errors/issues?